ui/js/reactrevisions.jsx

```diff
@@ -31,9 +35,35 @@ const RevisionItem = (props) => {
         tags += "backout";
     }
 
+    var tooltip = (<Tooltip id="tooltip">Copy revision to clipboard</Tooltip>);
```

Comment:
I think you could probably add the copied behaviour you want by making this a stateful component with a justCopied parameter. I don't think this should be too hard:

https://facebook.github.io/react/docs/state-and-lifecycle.html

i.e.

let tooltip = (<Tooltip id="tooltip">{ this.state.justCopied ? 'Copied' : 'Copy revision to clipboard' }</Tooltip>)

(I would also just inline the tooltip instead of creating a temporary variable for it)